en/docs/concepts/authorization/refresh-token-grant.md

```diff
@@ -0,0 +1,26 @@
+# Refresh Token Grant Type
+
+### Recommended Use
+
+The **refresh token grant** can be used when the current access token is expired or when a new access token is needed.
+With this grant type, the refresh token acts as a credential and is issued to the client by the authorization server.
+
+Issuing a refresh token is optional. If the authorization server issues a refresh token, it should come retrieve along
+with access token. WSO2 Identity Server issues refresh tokens for all other grant types other than the **implicit and 
+client credentials grant types**, as recommended by the OAuth 2.0 specification.
+
+### How does it work ?
+
+In the initial steps a refresh token has to be obtained before using it. This can be done using a grant type such as the
+authorization code or password grant type. Using the obtained refresh token, it can be obtained a new access token along
```

Comment:
it can be obtained a new access token along --> a new access token can be obtained along with....